TEveEventDisplay/src/TEveMu2eMCInterface.cc

```diff
@@ -99,86 +99,64 @@ namespace mu2e{
     TEveText *TEveMu2eMCInterface::GetLabel(int PDGCode, TEveMu2eCustomHelix *line, TEveMu2eCustomHelix *line_twoDXY, TEveMu2eCustomHelix *line_twoDXZ){
       const char* pid = "pid";
       auto t = new TEveText(pid);
+      Color_t color;
+      double posy = 0;
+      double posz = 3000.0;
       t->SetFontSize(15);
-      if(PDGCode == +11){ 
-        line->SetLineColor(kRed);//electrons
-        line_twoDXZ->SetLineColor(kRed);
-        line_twoDXY->SetLineColor(kRed);
-        pid = "electron -";
-        t->SetText(pid);
-        t->SetMainColor(kRed);
-        t->RefMainTrans().SetPos(0.0,400.0,30000.0);
```

Comment:
What are 0, 400, and 30000.  Is the 30,000 here the same in mm as the 3,000 above in cm?